src/plugins/KeyboardEvents.js

```diff
@@ -223,18 +223,18 @@ module.exports = class KeyboardEvents {
 
         if (!this.isListenerAdded) {
             // This avoid having continuously pressed keys because of alt+tab or any other command that blur from tab
-            window.addEventListener('blur', this.cancelAllPressedKeys.bind(this));
+            this.removeBlurListener = this.instance.addListener(window, 'blur', this.cancelAllPressedKeys.bind(this));
 
             if (!this.keyboardCallbacks) {
-                this.keyboardCallbacks = new Map([
-                    ['keypress', this.onKeyPress.bind(this)],
-                    ['keydown', this.onKeyDown.bind(this)],
-                    ['keyup', this.onKeyUp.bind(this)]
-                ]);
+                this.keyboardCallbacks = [
+                    {event: 'keypress', handler: this.onKeyPress.bind(this), removeListener: null},
+                    {event: 'keydown', handler: this.onKeyDown.bind(this), removeListener: null},
+                    {event: 'keyup', handler: this.onKeyUp.bind(this), removeListener: null}
+                ];
             }
             this.instance.root.focus();
-            this.keyboardCallbacks.forEach((value, key) => {
-                window.addEventListener(key, value);
+            this.keyboardCallbacks.forEach((item, index, array) => {
```

Comment:
I changed `keybaordCallbacks` to be an array instead of a map. This way I'm using the additionnal [`index` and `array` args](https://developer.mozilla.org/fr/docs/Web/JavaScript/Reference/Global_Objects/Array/forEach#array) in order to modify the array. The `foreach` method doesn't modify the array by itself, but the callback inside can. ([see paragrah right before the note](https://developer.mozilla.org/fr/docs/Web/JavaScript/Reference/Global_Objects/Array/forEach#description)).

See mini-demo too:
<img width="489" alt="image" src="https://github.com/Genymobile/genymotion-device-web-player/assets/69793750/684246a8-1cdb-4ae8-b1df-8fdfb9246820">